docs/src/main/asciidoc/features.adoc

```diff
@@ -8,18 +8,30 @@
 2016-02-02 15:31:01.936  INFO [bar,46ab0d418373cbc9,46ab0d418373cbc9,false] 23030 --- [nio-8081-exec-4] ...
 ----
 +
-(notice the `[appname,traceId,spanId,exportable]` entries from the MDC).
+(notice the `[appname,traceId,spanId,exportable]` entries from the MDC. `appname`, `traceId` , `spanId` seem to be quite
```

Comment:
if it isn't too much trouble, just enumerate all three vs assuming people know what they are. Particularly appname has been hard to pin down.

* spanId - the id of a specific operation that took place
* appname - the name of the application that logged the span
* traceId - the id of the latency graph that contains the span